src/main/java/org/folio/innreach/config/SpringAsyncConfig.java

```diff
@@ -47,13 +47,24 @@ public ThreadPoolTaskScheduler prepareScheduler() {
     return executor;
   }
 
-  @Bean("schedulerTaskExecutor")
-  public Executor schedulerTaskExecutor() {
+  @Bean("initialSchedulerTaskExecutor")
+  public Executor initialSchedulerTaskExecutor() {
     ThreadPoolTaskExecutor executor = new ThreadPoolTaskExecutor();
     executor.setCorePoolSize(schedulerTaskPoolSize);
     executor.setMaxPoolSize(schedulerTaskPoolSize + 10);
     executor.setQueueCapacity(50);
-    executor.setThreadNamePrefix("schedulerTaskExecutor-");
+    executor.setThreadNamePrefix("initialSchedulerTaskExecutor-");
+    executor.initialize();
+    return executor;
+  }
+
+  @Bean("ongoingSchedulerTaskExecutor")
+  public Executor ongoingSchedulerTaskExecutor() {
+    ThreadPoolTaskExecutor executor = new ThreadPoolTaskExecutor();
+    executor.setCorePoolSize(schedulerTaskPoolSize);
+    executor.setMaxPoolSize(schedulerTaskPoolSize + 10);
+    executor.setQueueCapacity(50);
```

Comment:
Should these be configurable? Are they configurable for initial contributions? Not saying they should be but maybe if they are for initial.

Reply:
I think what we had for initial is fine. If it was only `CorePoolSize` then that's probably good enough for now.